hazelcast-integration/spring-hibernate-2ndlevel-cache/README.md

```diff
@@ -37,13 +37,13 @@ In order to configure Hazelcast as second-level cache provider, you need to add
 <dependency>
     <groupId>com.hazelcast</groupId>
     <artifactId>hazelcast-hibernate53</artifactId>
-    <version>5.0.1-SNAPSHOT</version>
+    <version>5.2.0</version>
 </dependency>
 
 <dependency>
     <groupId>com.hazelcast</groupId>
     <artifactId>hazelcast</artifactId>
-    <version>5.4.0</version>
+    <version>5.5</version>
```

Comment:
why not 5.5.0?
best it matches whats in the POM?

Reply:
Ooops

5a884a0418709651fba21b7cd9458c116fff3b0e